### ISSUE TITLE
Publication of the plugin in packagist.org

### ISSUE THREAD

**Issue by Jazz-Man** [qtranslate/qtranslate-xt#775]
It would be very good if this plugin was published on the packagist. This would make it possible to install this plugin using the command 
```
composer require qtranslate/qtranslate-xt
```

I tried to add this plugin but the package will not be automatically updated. 
https://packagist.org/packages/qtranslate/qtranslate-xt

![Знімок екрана 2020-01-09 о 16 15 02](https://user-images.githubusercontent.com/6892898/72074811-42602a00-32fb-11ea-90d6-1265c00a25bb.png)

I can't do that because it's not my repository. It will take a few seconds for the author :)

**Comment by herrvigg** [comment]
Yes i know but i don't want to add maintenance effort and extra constraints in qTranslate which cannot even be pushed to wordpress.org in the current state. It will migrate some day.

Btw i'm aösp wondering if it makes much sense to push WP to packagist separately, how does it integrate in the WP installation? You have to activate it manually anyway?

**Comment by Juje** [comment]
I have the following config in my `composer.json` file:

```json
"extra":{
    "installer-paths": {
        "htdocs/content/plugins/{$name}/": ["type:wordpress-plugin"],
        "htdocs/content/mu-plugins/{$name}/": ["type:wordpress-muplugin"],
        "htdocs/content/themes/{$name}/": ["type:wordpress-theme"]
    },
    "wordpress-install-dir": {
        "johnpbloch/wordpress-core": "htdocs/cms"
    }
},
```

And that installs the plugin in the correct directory. Cause right now I have installed QtranslateXT as a git submodule which is ins't ideally but it works.

Ps. I'm using [Themosis](https://framework.themosis.com) and that's why the directory layout is different.

**Comment by herrvigg** [comment]
@Juje just be case, have you ever tried to install qTranslate-XT as a `mu-plugin`?
